core/templates/pages/exploration-editor-page/translation-tab/translation-tab.component.ts

```diff
@@ -0,0 +1,336 @@
+        );
+        // Toggles the translation tab tutorial on/off.
+        $scope.translationTutorial = false;
+        $scope.TRANSLATION_TUTORIAL_OPTIONS = [{
+          type: 'title',
+          heading: 'Translations In Oppia',
+          text: (
+            'Hello, welcome to Translation Tab! ' +
+            'This tour will walk you through the translation page. ' +
+            'Hit the "Next" button to begin.')
+        }, {
+          type: 'function',
+          fn: function(isGoingForward) {
+            $('html, body').animate({
+              scrollTop: (isGoingForward ? 0 : 20)
+            }, 1000);
+          }
+        }, {
+          type: 'element',
+          selector: _ID_TUTORIAL_TRANSLATION_LANGUAGE,
+          heading: 'Choose Language',
+          text: ('Start your translation by choosing the language that ' +
+            'you want to translate to.'),
+          placement: 'bottom'
+        }, {
+          type: 'function',
+          fn: function(isGoingForward) {
+            var idToScrollTo = (
+              isGoingForward ? _ID_TUTORIAL_TRANSLATION_OVERVIEW :
+              _ID_TUTORIAL_TRANSLATION_LANGUAGE);
+            $('html, body').animate({
+              scrollTop: angular.element(idToScrollTo).offset().top - 200
+            }, 1000);
+          }
+        }, {
+          type: 'element',
+          selector: _ID_TUTORIAL_TRANSLATION_OVERVIEW,
+          heading: 'Choose a Card to Translate',
+          text: ('Then, choose a card from the exploration overview by ' +
+          'clicking on the card. The selected card will have ' +
+          'a bolded border. Cards that have missing translations are ' +
+          'coloured yellow or red, and these are good places to start.'),
+          placement: 'left'
+        }, {
+          type: 'function',
+          fn: function(isGoingForward) {
+            var idToScrollTo = (
+              isGoingForward ? _ID_TUTORIAL_TRANSLATION_STATE :
+              _ID_TUTORIAL_TRANSLATION_OVERVIEW);
+            $('html, body').animate({
+              scrollTop: angular.element(idToScrollTo).offset().top - 200
+            }, 1000);
+          }
+        }, {
+          type: 'element',
+          selector: _ID_TUTORIAL_TRANSLATION_STATE,
+          heading: 'Choose a Part of the Card to Translate',
+          text: (
+            '<p>Next choose one of the parts of the lesson card from ' +
+            'menu at the top. This lists all the translatable parts ' +
+            'of the card. Within each tab, multiple sections may be ' +
+            'available for translating.</p>'),
+          placement: 'bottom'
+        }, {
+          type: 'function',
+          fn: function(isGoingForward) {
+            $('html, body').animate({
+              scrollTop: (isGoingForward ? 0 : 20)
+            }, 1000);
+          }
+        }, {
+          type: 'title',
+          heading: 'Recording Audio',
+          text: ('<p>To create audio translations in Oppia, ' +
```

Comment:
Shouldn't these things be tested?

Reply:
I don't think it's necessary because it's just string, there is no logic except for the `fn` property.

Reply:
Hmmm, ok